src/App.css

```diff
@@ -2,3 +2,20 @@
   /* text-align: center; */
 }
 
+.ant-list .ant-row > div,
```

Comment:
It would be great to have a comment why this is added (maybe with a link to the issue).

Reply:
it's based on the thread here: https://github.com/ant-design/ant-design/issues/40786 --> https://stackblitz.com/edit/react-wf9wmw-2i6mck?file=index.css,demo.tsx 
and some adaptions regards the used implementation with cover, body and footer

Reply:
and caused by the mentioned issue: https://github.com/toniebox-reverse-engineering/teddycloud/issues/162

Reply:
I mean to add this to the css itself, so we can remove the workaround if this is fixed in antd

Reply:
i am not sure if this is a bug at all in general. And ticket is also closed already. Won't expect any change in antd.

Reply:
added comment in app.css